package.json

```diff
@@ -27,7 +28,10 @@
     "tar": "^1.0.3"
   },
   "config": {
-    "excludeAbi": "11"
+    "excludeAbi": "11",
+    "oracleUrl": "http://7e9918db41dd01dbf98e-ec15952f71452bc0809d79c86f5751b6.r22.cf1.rackcdn.com",
+    "oracleVersion": "2.0.0",
+    "driverModule": "oracledb@^1.11.0"
```

Comment:
Now I don't think it is ideal to have the configuration in the installer module for the following reasons:

1. We should be able to use the same version of `loopback-oracle-installer` for different versions of `loopback-connector-oracle`.

2. With the proposed change, we'll have create a new release of the installer every time we need to update the connector.